recipe/meta.yaml

```diff
@@ -25,20 +24,21 @@ requirements:
     - numpy   1.20  # [not (osx and arm64) and py<310]
     - numpy   1.21  # [(osx and arm64) and py<310]
     - numpy   1.21  # [py==310]
+    - numpy   1.23  # [py==311]
     - pip
     - python
     - setuptools
     - wheel
   run:
-    - cython >=0.29,!=0.29.18
+    - cython >=0.29,!=0.29.18,!=0.29.31
```

Comment:
They are kind of used. There is https://github.com/alkaline-ml/pmdarima/blob/v2.0.3/pmdarima/utils/_show_versions.py#L13-L24, which is used to import the module at https://github.com/alkaline-ml/pmdarima/blob/v2.0.3/pmdarima/utils/_show_versions.py#L65.

I did look at these before creating the PR and thought we would probably want to keep the original behavior... Like if someone wants to open an issue, I'm guessing the package maintainer might request the user to print the versions?

But I don't mind removing them since the code will still work anyway.

Reply:
ok, I guess it's fine to leave it this way..